README.md

```
@@ -1,6 +1,25 @@
 # Raycast - Hacker News
 
+## Overview
+
+- This is an UNPUBLISHED extension for Raycast that shows the top stories from Hacker News.
+- Raycast is available for macOS now, and there is a waitlist for Windows.
+- This repository was initialized using the 'Create Extension' command in Raycast. Instructions for this can be found here: https://developers.raycast.com/basics/create-your-first-extension
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)